src/components/cylc/tree/Tree.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <InfiniteTree
+    ref="tree"
+    class-name="tree my-2"
+    :tree-data="treeData"
+    :tab-index="0"
+    :on-open-node="onOpenNode"
+    :on-close-node="onCloseNode"
+    >
+    <template slot-scope="{ node, tree }" slot="default">
+      <div class="node">
+        <v-layout class="node-data" :style="nodeStyle(node)" wrap v-if="node.__type === 'cyclepoint'">
+          <a @click="toggleClick($event, node, tree)" :class="togglerClass(node)"><span v-html="togglerContent(node)"></span></a>
+          <v-flex shrink>
+            <task :status="node.state.state" :progress=0 />
+          </v-flex>
+          <v-flex grow>
+            <span class="mx-1">{{ node.name }}</span>
+          </v-flex>
+        </v-layout>
+        <v-layout class="node-data" :style="nodeStyle(node)" wrap v-else-if="node.__type === 'family'">
```

Comment:
I don't know if it's possible to define these `<v-layout>` parts separately, would make this template easier to work with.

Reply:
Good idea. I will have a look later over the weekend or on Monday to see if it would be something easy to implement.

Reply:
Resolving as this was done in #396 . Added a **pending** section to the first/main text of this issue with an item for this.